ui/pages/confirmations/hooks/useGasFeeInputs.test.js

```diff
@@ -190,6 +193,17 @@ describe('useGasFeeInputs', () => {
   });
 
   describe('editGasMode', () => {
+    beforeEach(() => {
```

Comment:
Can we make sure adding a small unit test where it calls mocked `selectNetworkConfigurationByChainId` with given chain id?

Reply:
test added [here](https://github.com/MetaMask/metamask-extension/pull/28391/files#diff-9c8c03c352032e2a5051eb34891d9f417ef712f6cf39b5aa1c30fc98986c4681R193-R196)